backend/controllers/notes.ts

```diff
@@ -1,82 +1,83 @@
-import { Router, Request } from "express";
+import { Router } from "express";
 import Note from "../models/note";
 import User from "../models/user";
-import jwt from "jsonwebtoken";
+import middleware, { RequestWithUserId } from "../middleware";
+import convertNoteFromDatabaseToNoteToResponse from "../utils/convertNoteFromDatabaseToNoteToResponse";
 
 const notesRouter = Router();
 
-const getTokenFrom = (request: Request) => {
-	const authorization = request.get("authorization");
-	if (authorization && authorization.startsWith("Bearer ")) {
-		return authorization.replace("Bearer ", "");
-	}
-
-	return null;
-};
+export interface NoteResponse {
+	content: string;
+	important: boolean;
+	id: string;
+}
 
 notesRouter.get("/", async (_request, response) => {
 	const notes = await Note.find({});
-	response.json(notes);
+	const notesToReponse: NoteResponse[] = notes.map(
+		convertNoteFromDatabaseToNoteToResponse
```

Comment:
I think this name makes more sense
```suggestion
		convertNoteModelToNoteResponse
```